caikit/runtime/model_management/model_manager.py

```diff
@@ -222,22 +359,6 @@ def get_model_size(self, model_id) -> int:
             )
 
         loaded_model = self.loaded_models[model_id]
-
-        # Model sizes should all be cached
```

Comment:
This was code that would never be hit unless someone mutated `loaded_models` outside of `load_model` which should never happen except in a nefarious unit test.